employees_info.py

```diff
@@ -0,0 +1,60 @@
+
+import tables
+import options
+
+
+create_employees_info = """create table employees_info(
```

Comment:
maybe it makes sense to move this code to another place that is more related to working with tables?